docs/getting-started/3rd-party-bindings.md

```diff
@@ -0,0 +1,18 @@
+# 3rd Party Bindings
+
```

Comment:
I suggest the following sentence at the beginning:

Some organizations are maintaining their own wrappers and language bindings around the BDK libraries. While these are not maintained by the Foundation, they have users in production and are worth looking into!